### ISSUE TITLE
(NO LONGER NEEDED) Record form: Move "delete record" function to Record form

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#841]
# Story
Today, I can delete a record directly from the Browse > Records page. It should not be so easy for me to delete a record without even looking at it or considering the implications. I'd like to move the delete function out of the Browse view and into the individual record form -- just as we do when you want to delete an indicator or result.

# Affects
1. Record form: Adding a delete function with PNotify validation
2. Browse > Records view: Once the above item is done, we can remove the column of delete buttons.

# Acceptance criteria

## Record form
- [ ] At the bottom of the Record form, add a "Delete this record" button. 

Until we implement #831, the button should behave as it does today -- taking you to the full page confirmation dialogue.

![image](https://user-images.githubusercontent.com/33670923/49121289-be68f100-f264-11e8-8134-d1a5cc526efb.png)

## Browse > Records view
- [ ] Remove the column of "Delete" buttons. Retain the column of "Edit" buttons.

![image](https://user-images.githubusercontent.com/33670923/49120972-b197cd80-f263-11e8-8380-571f329797ef.png)

**Comment by jennyhavah** [comment]
Since documents will only be linked to Projects going forward, and we are no longer making improvements that only benefit Projects, I'm closing the ticket.
